client/src/utils/optionalQueryString.ts

```diff
@@ -1,3 +1,3 @@
-export function optionalQueryString(value: string | undefined) {
+export function optionalQueryString(value: string | string[] | undefined) {
   return value ? String(value) : undefined;
```

Comment:
if value could be an array, shouldn't return value be adjusted or it would work fine as it is?